src/components/Separator/Separator.tsx

```diff
@@ -1,42 +1,34 @@
 import * as React from "react";
-import { getClassName } from "../../helpers/getClassName";
 import { classNames } from "../../lib/classNames";
-import { usePlatform } from "../../hooks/usePlatform";
 import "./Separator.css";
 
 export interface SeparatorProps extends React.HTMLAttributes<HTMLDivElement> {
   /**
    * С этим свойством компонент не будет иметь отступы слева и справа
    */
   wide?: boolean;
+
+  /**
+   * @deprecated Это свойство устарело и будет удалено в 5.0.0. Используйте [`Spacing`](#/Spacing).
+   */
   expanded?: boolean;
 }
 
-const Separator: React.FC<SeparatorProps> = ({
-  wide,
-  expanded,
-  ...restProps
-}) => {
-  const platform = usePlatform();
-
-  return (
+export const Separator: React.FC<SeparatorProps> = React.memo(
+  ({ wide, expanded, ...restProps }) => (
     <div
       {...restProps}
       aria-hidden="true"
-      // eslint-disable-next-line vkui/no-object-expression-in-arguments
-      vkuiClass={classNames(getClassName("Separator", platform), {
-        "Separator--wide": wide,
-      })}
+      vkuiClass={classNames("Separator", wide && "Separator--wide")}
```

Comment:
```suggestion
      vkuiClass={classNames("Separator", !wide && "Separator--padded")}
```

Тогда мы сможем в стилях избавиться от `:not`. Что скажешь?

Reply:
- добавил `Separator--padded`
- `Separator--wide` оставил, выпилим в v5